### PR TITLE
Course Unit Enrollment view for users - will display a user's progres…

### PR DESCRIPTION
**Pull request by mehalshah**
…s for a given unit

### CODE REVIEW

**Comment by mehalshah** [comment]
Here's what the course unit view looks like now - both assigned modules show up side by side with their relevant icons. Additionally, all resources that could possibly have been associated with the course unit depending on assessments are available to the user at the bottom.

image

File: dashboard/app/views/plc/enrollment_unit_assignments/show.html.haml [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+        = link_to task_assignment.plc_task.name, plc_enrollment_task_assignment_path(task_assignment)
+        %br
 
-= link_to 'Edit', edit_plc_enrollment_unit_assignment_path(@plc_enrollment_unit_assignment)
-\|
-= link_to 'Back', plc_enrollment_unit_assignments_path
+.all_possible_resources
+  %h4
+    All learning resources
+  - @enrollment_unit_assignment.plc_course_unit.plc_evaluation_questions.map(&:plc_evaluation_answers).flatten.map(&:plc_learning_module).uniq.delete_if{|x| x.nil?}.map(&:plc_tasks).flatten.select!{|x| x.class == Plc::LearningResourceTask}.each do |x|
```
Comment (aoby) [inline dashboard/app/views/plc/enrollment_unit_assignments/show.html.haml]: This is a pretty complex query in the view. Why not move it into the model and add a unit test?

**Comment by aoby** [comment]
LGTM, aside from above refactor suggestion

File: dashboard/app/views/plc/enrollment_unit_assignments/show.html.haml (hunk 1 above)
Comment (mehalshah) [inline dashboard/app/views/plc/enrollment_unit_assignments/show.html.haml]: Makes sense, will do

**Pull request merged**